feat(components): create navigation sider

Hellomyguest opened the pull request:
add pagelayout component with navigation sider

META DATA: issue39

Commit:
feat(components): create navigation sider

add pagelayout component with navigation sider

META DATA: issue39

Commit:
feat(components): pageLayout component updated

sider moved to separate component
theme switcher moved to sider
student nav moved to page header

META DATA: issue39

dpolevodin commented on src/pages/sign/signIn/page.tsx:
```diff
@@ -30,64 +30,59 @@ export const SignInPage = () => {
+                ) {
+                  return Promise.resolve();
+                }
+                return Promise.reject(new Error("Не верный пароль"));
```
орфографическая ошибка

Hellomyguest replied:
fixed )

Commit:
fix(components): fixed error message

META DATA: issue39

Commit:
feat(components): change theme button beside icon

add change theme button for collapsed sider beside theme icon

META DATA: issue39
